adding docs change log (#1318)
temporalio/documentation: change-log/july-release-2022.md

---
slug: july-release-2022
title: July 25 2022
tags:
  - docs-change-log
date: 2022-07-25T00:00:00Z
---

:::success THANK YOU!

A **HUGE** thank you to everyone who participated in our user testing sessions and responded to our [user survey](https://forms.gle/VWgQdPbV5qa8zJVBA).

Your feedback is invaluable to us as we work to improve the information that is available to Temporal users.

:::

- Python examples are now available in much of the Foundations section of the Application Development guide.

> - An Observability section has been added to the Application development guide.

Reviewer: ```suggestion
- We added an Observability section to the Application Development guide.
```
- Need to init-cap "Development," I'm fairly sure. If not, need to lowercase it in the preceding item.
- Switching to active voice is per style guides, but it's not a big thing here.

> - Cloud release notes are now available at [docs.temporal.io/cloud/release-notes](https://docs.temporal.io/cloud/release-notes/).

Reviewer: ```suggestion
- Temporal Cloud release notes are now available at [docs.temporal.io/cloud/release-notes](https://docs.temporal.io/cloud/release-notes/).
```
